articles/aks/supported-kubernetes-versions.md

```diff
@@ -128,6 +130,38 @@ New Supported Version List
 1.17.*9*, 1.17.*8*, 1.16.*11*, 1.16.*10*
 ```
 
+## Platform Support Policy
+For one GA minor version after the Kubernetes support policy, AKS provides platform support. The platform support window of Kubernetes versions on AKS is known as "N-3".
+
+During platform support, customers running N-3 versions will only receive support from Microsoft for AKS platform related issues. Any issues related to the upstream Kubernetes community will not be supported. 
+
+This table outlines suppport guidelines for Community Support compared to platform support.
+
+| Support category | Community Support (N-2) | Platform Support (N-3) | 
+|---|---|---|
+| Kubernetes components (including Add-ons) | Supported | Not supported|
+| Component updates | Supported | Not supported|
+| Component hotfixes | Supported | Not supported|
+| Applying bug fixes | Supported | Not supported|
+| Applying security patches | Supported | Not supported|
+| Kubernetes API support | Supported | Not supported|
+| Platform (Azure) availability | Supported | Supported|
+| Infrastructure SLA| Supported | Supported|
+| Platform (AKS) SLA| Supported | Not supported|
+|Control Plane SLA| Supported | Supported|
+| Cluster or node pool creation| Supported | Not supported|
+| Node pool scaling| Supported | Supported|
+| VM availability| Supported | Supported|
+| Storage, Networking related issues| Supported | Supported with the exception of bug fixes and retired components |
+| Upgrades from N-3 to a supported version| Supported | Supported|
+| Start/stop | Supported | Supported|
+| Rotate certificates | Supported | Supported|
+| Node pool snapshot| Supported | Not supported|
+| Node image upgrade| Supported | Not supported|
+
+ > [!NOTE]
+  > The above table is subject to change and outlines common support scenarios. Any scenarios including kubernetes components, fixes, or retirements will not be supported for N-3.
```

Comment:
we should probably put a not they can ask for further clarification by opening a docs issue or support ticket.